tests/Model/Customer/CustomerListTest.php

```diff
@@ -643,16 +774,50 @@ public function testCombinedSearchQueryFilter()
         $modifiedListing = $handler->getListing();
         $this->assertEquals(1, $modifiedListing->getCount());
 
+        // -- Tests for Exact Search (Field should must match exactly)
+
+        $listing = new Customer\Listing();
+        $handler = new FilterHandler($listing);
+
+        $searchFilter1 = new SearchQuery(['firstname'], '"jane"');
+        $searchFilter2 = new SearchQuery(['firstname'], '"john"');
+        $handler->addFilter (new BoolCombinator([$searchFilter1, $searchFilter2], 'OR'));
+
+        $modifiedListing = $handler->getListing ();
+        $this->assertEquals (2, $modifiedListing->getCount ());
+
         // --------------------
         $listing = new Customer\Listing();
         $handler = new FilterHandler($listing);
 
-        $searchFilter1 = new SearchQuery(['lastname'], '*hugo*');
-        $searchFilter2 = new SearchQuery(['firstname'], 'pet*');
-        $handler->addFilter(new BoolCombinator([$searchFilter1, $searchFilter2], 'AND'));
+        $searchFilter1 = new SearchQuery(['firstname'], '"jane"');
+        $searchFilter2 = new SearchQuery(['firstname'], '"john"');
+        $handler->addFilter (new BoolCombinator([$searchFilter1, $searchFilter2], 'AND'));
 
-        $modifiedListing = $handler->getListing();
-        $this->assertEquals(1, $modifiedListing->getCount());
+        $modifiedListing = $handler->getListing ();
+        $this->assertEquals (0, $modifiedListing->getCount ());
+
+        // --------------------
+        $listing = new Customer\Listing();
+        $handler = new FilterHandler($listing);
+
+        $searchFilter1 = new SearchQuery(['lastname'], '"hugo"');
+        $searchFilter2 = new SearchQuery(['firstname'], '"peter"');
+        $handler->addFilter (new BoolCombinator([$searchFilter1, $searchFilter2], 'OR'));
+
+
+        $modifiedListing = $handler->getListing ();
+        $this->assertEquals (1, $modifiedListing->getCount ());
+
+        // --------------------
+        $listing = new Customer\Listing();
+        $handler = new FilterHandler($listing);
+
+        $searchFilter1 = new SearchQuery(['lastname'], '"hugo"');
+        $searchFilter2 = new SearchQuery(['firstname'], '"peter"');
```

Comment:
Can we also add a test where we mix `""` operator with `*` one?

Reply:
Yes, will do